src/components/playback/playerSelectionMenu.js

```diff
@@ -1,320 +1,325 @@
-define(['appSettings', 'events', 'browser', 'loading', 'playbackManager', 'appRouter', 'globalize', 'apphost'], function (appSettings, events, browser, loading, playbackManager, appRouter, globalize, appHost) {
-    'use strict';
+import appSettings from 'appSettings';
+import events from 'events';
+import browser from 'browser';
+import loading from 'loading';
+import playbackManager from 'playbackManager';
+import appRouter from 'appRouter';
+import globalize from 'globalize';
+import appHost from 'apphost';
 
-    function mirrorItem(info, player) {
+function mirrorItem(info, player) {
 
-        var item = info.item;
+    var item = info.item;
 
-        playbackManager.displayContent({
+    playbackManager.displayContent({
 
-            ItemName: item.Name,
-            ItemId: item.Id,
-            ItemType: item.Type,
-            Context: info.context
-        }, player);
-    }
+        ItemName: item.Name,
+        ItemId: item.Id,
+        ItemType: item.Type,
+        Context: info.context
+    }, player);
+}
 
-    function mirrorIfEnabled(info) {
+function mirrorIfEnabled(info) {
 
-        if (info && playbackManager.enableDisplayMirroring()) {
+    if (info && playbackManager.enableDisplayMirroring()) {
 
-            var getPlayerInfo = playbackManager.getPlayerInfo();
+        var getPlayerInfo = playbackManager.getPlayerInfo();
 
-            if (getPlayerInfo) {
-                if (!getPlayerInfo.isLocalPlayer && getPlayerInfo.supportedCommands.indexOf('DisplayContent') !== -1) {
-                    mirrorItem(info, playbackManager.getCurrentPlayer());
-                }
+        if (getPlayerInfo) {
+            if (!getPlayerInfo.isLocalPlayer && getPlayerInfo.supportedCommands.indexOf('DisplayContent') !== -1) {
+                mirrorItem(info, playbackManager.getCurrentPlayer());
             }
         }
     }
+}
 
-    function emptyCallback() {
-        // avoid console logs about uncaught promises
-    }
+function emptyCallback() {
+    // avoid console logs about uncaught promises
+}
 
-    function getTargetSecondaryText(target) {
+function getTargetSecondaryText(target) {
 
-        if (target.user) {
-
-            return target.user.Name;
-        }
+    if (target.user) {
 
-        return null;
+        return target.user.Name;
     }
 
-    function getIcon(target) {
+    return null;
```

Comment:
```suggestion
return target.user ? target.user.Name : null;
```
this complete function can be donel like this